docs/index.rst

```diff
@@ -1,26 +1,16 @@
-Toil
-====
-
-Toil is an open-source pure-Python workflow engine that lets people write better
-pipelines. You can:
-
-* Write your workflows in `Common Workflow Language`_ (CWL)
-* Run workflows on your laptop or on huge commercial clouds such as
-  `Amazon Web Services`_ (including the `spot market`_), `Microsoft Azure`_,
-  `OpenStack`_, and `Google Compute Engine`_
-* Take advantage of high-performance computing environments with batch systems
-  like `GridEngine`_, `Apache Mesos`_, and `Parasol`_
-* Run workflows concurrently at scale using thousands of nodes, managed by Toil's :ref:`Autoscaling` capabilities
-* Execute workflows efficiently with caching and resource requirement specifications
-* Easily link databases and services
-
-Toil is, admittedly, not quite as good as sliced bread, but it's about as close
-to it as you're gonna get.
-
-Check out our `website`_ for a more comprehensive list of Toil's features, read
-our `paper`_ to learn more about what Toil can do in the real world, or jump in
-and start with the :ref:`installation-ref` section. (Feel free to also join us
-on `GitHub`_ and `Gitter`_.)
+Toil Documentation
+==================
+
+Everything you need to know about Toil.
+
+* Toil is a scalable, efficient, cross-platform pipeline management system.
+* Run it on Amazon Web Services and Toil can automatically manage the number of workers.
+* Write your workflows with an easy to use Python API (Turing complete!) or CWL.
```

Comment:
@mr-c We are going to remove the description and rely on the web page (http://toil.ucsc-cgl.org/) to describe the features. I added the link there. Any suggestions for the web page are also welcome.